SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -64,6 +64,14 @@ DrawGrid {
 	vertGrid_ { arg g;
 		y.grid = g;
 	}
+	tickSpacing_ { arg val;
```

Comment:
Because this doesn't set the tick spacing, but rather a threshold below which tick behavior changes, should this be `minTickSpacing`?